src/main/java/uk/gov/digital/ho/hocs/queue/complaints/ComplaintService.java

```diff
@@ -28,67 +31,109 @@ public class ComplaintService {
     private final CaseworkClient caseworkClient;
     private final ClientContext clientContext;
     private final DocumentS3Client documentS3Client;
+    private final MessageLogService messageLogService;
 
     @Autowired
     public ComplaintService(WorkflowClient workflowClient,
                             CaseworkClient caseworkClient,
                             ClientContext clientContext,
-                            DocumentS3Client documentS3Client) {
+                            DocumentS3Client documentS3Client,
+                            MessageLogService messageLogService) {
         this.workflowClient = workflowClient;
         this.caseworkClient = caseworkClient;
         this.clientContext = clientContext;
         this.documentS3Client = documentS3Client;
+        this.messageLogService = messageLogService;
     }
 
     public void createComplaint(ComplaintData complaintData, ComplaintTypeData complaintTypeData) {
+        var documentSummary = createDocument(complaintData);
+        var caseUuid = createCase(complaintData, complaintTypeData, documentSummary);
+        var stageForCaseUUID = getCaseStage(caseUuid);
+        updateCaseUser(caseUuid, stageForCaseUUID);
```

Comment:
The code change around this is the [BREAKING] change, it allows us to properly know and investigate partial cases, rather than ignoring them and proceeding anyway.